sdk/cdsclient/client_queue.go

```diff
@@ -50,101 +52,86 @@ func shrinkQueue(queue *sdk.WorkflowQueue, nbJobsToKeep int) time.Time {
 }
 
 func (c *client) QueuePolling(ctx context.Context, jobs chan<- sdk.WorkflowNodeJobRun, pbjobs chan<- sdk.PipelineBuildJob, errs chan<- error, delay time.Duration, graceTime int, modelType string, ratioService *int, exceptWfJobID *int64) error {
-	t0 := time.Unix(0, 0)
 	jobsTicker := time.NewTicker(delay)
-	pbjobsTicker := time.NewTicker(delay * 10)
-	oldJobsTicker := time.NewTicker(delay * 5)
+	pbjobsTicker := time.NewTicker(delay * 2)
+
+	// This goroutine call the SSE route
+	chanSSEvt := make(chan SSEvent)
+	sdk.GoRoutine("RequestSSEGet", func() {
+		for ctx.Err() == nil {
+			time.Sleep(5 * time.Second)
```

Comment:
Sleep should be at the end of the loop, if ctx.Err() != nil in that delay a useless SSE request will be prepared.